test/Worker.Extensions.DurableTask.Analyzers.Test/Verifiers/CSharpVerifierHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using System;
+using System.Collections.Immutable;
+
+namespace Worker.Extensions.DurableTask.Analyzers.Tests
+{
+    internal static class CSharpVerifierHelper
+    {
+        /// <summary>
+        /// By default, the compiler reports diagnostics for nullable reference types at
+        /// <see cref="DiagnosticSeverity.Warning"/>, and the analyzer test framework defaults to only validating
+        /// diagnostics at <see cref="DiagnosticSeverity.Error"/>. This map contains all compiler diagnostic IDs
+        /// related to nullability mapped to <see cref="ReportDiagnostic.Error"/>, which is then used to enable all
+        /// of these warnings for default validation during analyzer and code fix tests.
+        /// </summary>
+        internal static ImmutableDictionary<string, ReportDiagnostic> NullableWarnings { get; } = GetNullableWarningsFromCompiler();
+
+        private static ImmutableDictionary<string, ReportDiagnostic> GetNullableWarningsFromCompiler()
```

Comment:
How is the `CSharpVerifierHelper` used in our scenario with the .NET Isolated Analyzer? Do we want to make all nullable reference types diagnostics an error instead of a warning? Do we need this verifier helper or can we remove it?

Reply:
Answered below (along with the other comments :))

Reply:
Thanks! How are we using this file? Is there any reason that we need to make nullable reference types diagnostics an error instead of a warning?

Reply:
This was part of the code generated, I assuming they would like to simulate how the compiler used by final users behave. For our case, we don't need that behavior (because we have full control of the written tests and we aren't looking for this kind of compilation errors). So, I am going to remove this additional configuration. If needed later, we can enable it again.